services/metadata_service/server.py

```diff
@@ -18,12 +18,12 @@
 PATH_PREFIX = os.environ.get("PATH_PREFIX", "")
```

Comment:
change default to `None` instead of empty string to match the default value for prefix in `app()` ?